stripe/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -103,11 +103,12 @@ internal class StripePaymentController internal constructor(
         } ?: PaymentAuthWebViewStarter.Legacy(host)
     }
 
-    private val stripe3ds2CompletionStarterFactory = { host: AuthActivityStarter.Host, requestCode: Int ->
-        stripe3ds2ChallengeLauncher?.let {
-            Stripe3ds2CompletionStarter.Modern(it)
-        } ?: Stripe3ds2CompletionStarter.Legacy(host, requestCode)
-    }
+    private val stripe3ds2CompletionStarterFactory =
+        { host: AuthActivityStarter.Host, requestCode: Int ->
```

Comment:
`host` needs to be on a new line, no?

Reply:
I think the rules for lambdas are slightly different